docs/Linux-CI.md

```diff
@@ -0,0 +1,27 @@
+# Linux CI in `homebrew/core`
+
+We currently use Ubuntu 16.04 for bottling in `homebrew/core`.
+
+### Ubuntu versus other distributions
+
+As of 2022, around 77% of our users are using Ubuntu. This is the reason why we have chosen this distribution for our base CI image.
+We have been using Ubuntu for CI since version 14.04 with success.
+The LTS versions have 5 year support. A new LTS version is released every 2 years.
+
+Our bottles are compatible with other distributions like Debian/CentOS, even when compiled on Ubuntu.
+
+### Next versions
+
+We are currently moving our CI to Ubuntu 20.04. This work will probably be done before end of 2022.
+Moving from Ubuntu 16.04 to Ubuntu 20.04 (and thus skipping 18.04) took more time than expected due to other more urgent issues we had to take care of.
```

Comment:
I thought we were doing 22.04 given glibc 2.35?

Reply:
Oh, yes, I misread this and have this question too.

Reply:
The majority of our users are using Ubuntu 20.04 and Glibc 2.31. I suggest that we build bottles using Ubuntu 20.04 and Glibc 2.31. 79% of our users (23% on Glibc 2.35 and 56% on Glibc 2.31) will be able to use their host's Glibc and compiler without needing to installed the brewed Glibc and GCC.